webpack.config.dist.js

```diff
@@ -0,0 +1,23 @@
+require("babel/register");
```

Comment:
I fixed this case and will try and fix other places as well, thanks @aduth for your input!